### PR TITLE
Update map developer permissions

### PR DESCRIPTION
**Pull request by Palmidence**

### CODE REVIEW

**Comment by amactus** [comment]
Mapdevs should still have broadcast and team alias - maybe give too?

**Comment by Palmidence** [comment]
@amactus I don't really see why they need to change team names in public, and I intentionally got rid of the give command because it can change the outcome of public matches.

File: models/groups/map_developers.yml [hunk 1]
```diff
@@ -87,6 +87,15 @@ minecraft_permissions:
   - commandbook.gamemode.change.other
```
Comment (Palmidence) [inline models/groups/map_developers.yml]: Updated so changes are made on beta/alpha, not mapdev

**Comment by mitchiiii** [comment]
The thing is world edit is also useful for events as well -- those permissions have been in this group for as long as I can remember. Not only that but it has been vital to hot fixing physical map issues during public matches, particularly when players get stuck in areas that they shouldn't be in. I understand you want to fix the ridiculous amount of permission Map Developers have, but this isn't exactly the right way. You need to look at disconnecting `/server mapdev` from the production environment so that our operator permissions on that server can't influence the wider network.

File: models/groups/map_developers.yml [hunk 2]
```diff
@@ -159,20 +153,9 @@ minecraft_permissions:
...
-  - worldedit.clipboard.*
   - worldedit.drain
   - worldedit.extinguish
-  - worldedit.history.*
-  - worldedit.navigation.*
-  - worldedit.region.*
-  - worldedit.remove
-  - worldedit.selection.*
-  - worldedit.tool.info
   - worldedit.wand
```
Comment (cswhite2000) [inline models/groups/map_developers.yml]: These changes would give mapdevs a wand but not let them used it for anything. Drain and extinguish don't require a wand, so you should probably just remove the wand permissions

**Comment by PotatoStealer** [comment]
consider giving mapdevs rotcon perms.

**Comment by Palmidence** [comment]
@PotatoStealer What is the permission for that?

**Comment by PotatoStealer** [comment]
pgm.rotation.set

**Comment by JexeI** [comment]
`   - commandbook.speed.flight
   - commandbook.speed.flight.other
   - commandbook.speed.walk
   - commandbook.speed.walk.other`
This should be added to public servers

**Comment by PotatoStealer** [comment]
observers don't really need this @Jexel

**Comment by amactus** [comment]
#32 now includes what's needed. Goodnight

**Pull request closed without merging**